ui/src/app/lib/atlasmap-data-mapper/services/mapping-management.service.ts

```diff
@@ -694,7 +700,7 @@ export class MappingManagementService {
    * @param position
    * @param offset
    */
-  private updateTransition(mapping: MappingModel, position?: string, offset?: number): void {
+  updateTransition(mapping: MappingModel, position?: string, offset?: number): void {
```

Comment:
For stubbing...

Reply:
Does adding `<any>` help to get around it?
https://stackoverflow.com/questions/8475564/using-jasmines-spyon-upon-a-private-method

Reply:
It did the trick. Thanks!